components/Releases/ReleaseLayout.jsx

```diff
@@ -20,67 +20,103 @@ export default function ReleaseLayout({
 }) {
   const [authorized, setAuthorized] = useState(!release.is_password_protected)
   const [releaseDate, setReleaseDate] = useState(new Date(release.release_date))
+  const [isClient, setIsClient] = useState(false)
 
   const sanitizedAbout = sanitize(release.about)
 
+  useEffect(() => {
+    setIsClient(true)
+
+    // if (
+    //   release.player_embed &&
+    //   release.player_embed.includes("album=") &&
+    //   !albumCode
+    // ) {
+    //   let embedArray = release.player_embed.split("/")
+
+    //   embedArray.forEach((value) => {
+    //     if (value.match(/(?:album=)\d+/)) {
+    //       setAlbumCode(value.slice(6))
+    //     }
+    //   })
+    // }
+  }, [])
+
```

Comment:
Remove this code if it's no longer needed

Reply:
gone